src/Session/profile.py

```diff
@@ -71,6 +71,14 @@ def addFoods(self, foods):
             return 1
         return 0
 
+    ''' Removes array of foods from the opened food storage that the user is in.  
+        Can remove multiple food items. '''
+    def removeFoods(self, foods):
```

Comment:
I think instead, you should have this return True/False based on if `self._opened_foodStorage.removeFoods(foods)` is successful or not. Which means you should also update what FoodStorage.removeFoods() returns. (And comments)